### PR TITLE
Add Migration For Recommendation Requests

### PR DESCRIPTION
**Pull request by basperheim1**
Closes https://github.com/ucsb-cs156-f24/proj-rec-f24-05/issues/64

Should be merged after https://github.com/ucsb-cs156-f24/proj-rec-f24-05/pull/63

Adds a migration file for the RecommendationRequest table

### CODE REVIEW

File: src/main/java/edu/ucsb/cs156/rec/controllers/RequestTypesController.java [hunk 1]
```diff
@@ -101,11 +101,11 @@ public RequestType postRequestType(
     }
 
     /**
-     * Update a single requestType
+     * Update a single request type
      * 
-     * @param id         id of the requestType to update
-     * @param incoming   the new RequestType
-     * @return           the saved requestType
+     * @param id  the name of the request type
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/rec/controllers/RequestTypesController.java:106]: No, you had it right the first time.  It's the id of the request type. Note that it's a Long value.
```suggestion
     * @param id  the id of the request type 
```

File: src/main/resources/db/migration/changes/RecommendationRequests.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                        "primaryKey": true,
+                        "primaryKeyName": "RECOMMENDATIONREQUESTS_PK"
+                      },
+                      "name": "ID",
+                      "type": "BIGINT"
+                    }
+                  },
+                  {
+                    "column": {
+                      "name": "PROFESSOR",
```
Comment (pconrad) [inline src/main/resources/db/migration/changes/RecommendationRequests.json:38]: I think this needs to be 
```suggestion
                      "name": "PROFESSOR_ID",
```

File: src/main/resources/db/migration/changes/RecommendationRequests.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                      "type": "BIGINT"
+                    }
+                  },
+                  {
+                    "column": {
+                      "name": "REQUESTER",
```
Comment (pconrad) [inline src/main/resources/db/migration/changes/RecommendationRequests.json:44]:
```suggestion
                      "name": "REQUESTER_ID",
```

**Review by pconrad — CHANGES_REQUESTED** [review]
A couple of things...

File: src/main/resources/db/migration/changes/RecommendationRequests.json (hunk 2 above)
Comment (basperheim1) [reply src/main/resources/db/migration/changes/RecommendationRequests.json]: Fixed

File: src/main/resources/db/migration/changes/RecommendationRequests.json (hunk 3 above)
Comment (basperheim1) [reply src/main/resources/db/migration/changes/RecommendationRequests.json]: Fixed

File: src/main/java/edu/ucsb/cs156/rec/controllers/RequestTypesController.java (hunk 1 above)
Comment (basperheim1) [reply src/main/java/edu/ucsb/cs156/rec/controllers/RequestTypesController.java]: Fixed

**Review by RyanZanone — APPROVED** [review]
LGTM

**Review by pconrad — DISMISSED** [review]
LGTM

**Pull request merged**